src/info.cc

```diff
@@ -0,0 +1,29 @@
+#include "info.h"
+
+Info::Info() {
+  set_hexpand(false);
+  set_halign(Gtk::Align::ALIGN_END);
+  
+  auto content_area=dynamic_cast<Gtk::Container*>(get_content_area());
+  label.set_max_width_chars(40);
+  label.set_line_wrap(true);
+  content_area->add(label);
+    
+  auto provider = Gtk::CssProvider::create();
+  provider->load_from_data("* {border-radius: 5px;}");
+  get_style_context()->add_provider(provider, GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
+}
+
+void Info::print(const std::string &text) {
+  if(!enabled)
```

Comment:
@zalox We could change the preferences like this maybe:
```json
{
    "version": "1.1.3-1",
    "gtk_theme": {
        "name_comment": "Use \"\" for default theme, At least these two exist on all systems: Adwaita, Raleigh",
        "name": "",
        "variant_comment": "Use \"\" for default variant, and \"dark\" for dark theme variant. Note that not all themes support dark variant, but for instance Adwaita does",
        "variant": ""
    },
    "window": {
        "default_size": {
            "width": "1024",
            "height": "768"
        },
        "show": {
          "directories": true,
          "terminal": true,
          "source_map": true,
          "info_messages": true,
          "info_messages_dealy": 3,
        }
    },
    "terminal": {
        "history_size": "1000",
        "font_comment": "Use \"\" to use source.font with slightly smaller size",
        "font": ""
    },
    ...
}
```
But let's consider this after this PR. 

At the moment, we can't have "hidden" preference items. Preference items not found in files.h will be removed when version gets bumped. This is needed for us to be able to change the preference item structure between versions, and users not to have unused nodes after upgrading juCi++. Adding "hidden" preference items would require us to add exceptions to this rule.